CODEOWNERS

```diff
@@ -22,6 +22,7 @@
 
 /Workbooks/WaasUpdateInsights/** @microsoft/update-compliance-workbooks
 /Workbooks/Azure*Security*Center/** @microsoft/azure-security-center-workbooks
+/Workbooks/Defender*For*IoT/** @microsoft/defender-for-iot-workbooks
```

Comment:
you'll also want to add a separate entry lower down for your gallery file as well, so that you own that too, since it isn't inside the workbooks folder

Reply:
Make sense.
Thanks!

Reply:
Added!